README.md

```diff
@@ -93,14 +99,19 @@ This URL can be used in Session to join the group
 
```

Comment:
Above this, the URL displayed contains a live IP (and probably a live pubkey).  We should use an RFC example IP instead, and a bogus pubkey.

Perhaps: `URL: http://192.0.2.123/fishing?public_key=0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef`